website/docs/reference/resource-configs/snowflake-configs.md

```diff
@@ -9,6 +9,392 @@ To-do:
 - use the reference doc structure for this article / split into separate articles
 --->
 
+## Iceberg table format <Lifecycle status="beta"/>
+
+Snowflake Iceberg tables support for the dbt-snowflake adapter is available for three out-of-the-box materializations:
```

Comment:
```suggestion
Iceberg table format support on the dbt-snowflake adapter is available for three out-of-the-box materializations:
```

This sentence feels really awkward when you read it out loud. I tried to correct it but I don't think I'm quite there yet.